pypeit_test

```diff
@@ -108,9 +112,42 @@ def main():
         # Setups
         setups = next(os.walk(os.path.join(raw_data, instr)))[1]
 
+        # Cut me down
         if pargs.debug and ('blue' not in instr):
             continue
 
+        # Develop cut
+        if pargs.tests == 'develop':
+            if instr == 'Keck_DEIMOS':
```

Comment:
Typo here, I think you mean 600ZD_M_6500

Reply:
You might consider adding 830G_M_8100_26, since these are big manufactured slits on different detectors. 

The other thing you might consider is whether we need to always reduce red/blue detectors for a given setup. We are currently doing that in the PypeIt files. These do test different things of course.